src/components/Book.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Book() {
+  return (
+    <div>
+      <h1>Machine learning by Chime</h1>
+    </div>
+  );
+}
```

Comment:
- [ ] Please, consider that here for a simple book component you have to implement the following:
* Takes the book data as a prop and renders the title and the author
* Has a button for removing books
  * the button wouldn't remove the books *yet*, this is just a placeholder for now